### PR TITLE
fix: #2974 Simplify puts plus and minus signs next to eachother

### PR DESCRIPTION
**Pull request by MaybePixem**
Fix for #2974
Tests included
Added new rule to remove + before - when the term is in parantheses

### CODE REVIEW

File: src/function/algebra/simplify.js [hunk 1]
```diff
@@ -425,7 +425,9 @@ export const createSimplify = /* #__PURE__ */ factory(name, dependencies, (
       assuming: { multiply: { associative: true } }
     },
 
-    { l: 'n1/(-n2)', r: '-n1/n2' }
+    { l: 'n1/(-n2)', r: '-n1/n2' },
+
+    { l: 'n+-(n1)', r: 'n-(n1)' }
```
Comment (josdejong) [inline src/function/algebra/simplify.js:430]: 😎  so simple. 

I somehow expected this case to be handled by the rule `{ l: 'n+-n1', r: 'n-n1' }` at line 394. Can you put this new rule right under line 394 to keep these two similar rules grouped together?

**Review by josdejong — CHANGES_REQUESTED** [review]
Thanks @MaybePixem, this looks good. I made one small remark, can you have a look at that?

File: src/function/algebra/simplify.js (hunk 1 above)
Comment (MaybePixem) [reply src/function/algebra/simplify.js]: sure

Comment (josdejong) [reply src/function/algebra/simplify.js]: 👍

**Review by josdejong — APPROVED** [review]
Thanks for the update, I'll merge your fix now.

**Pull request merged**